### ISSUE TITLE
[Rendering] iOS App crashing when rendering Adaptive Card

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#6114]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/7438 

**Details from original post:**
### Target Platforms

iOS

### SDK Version

v1.0

### Application Name

Microsoft Teams

### Problem Description

Created Card as a response from custom Chatbot. The card renders just fine within MS Teams on MacOS.
However, every time the Chat with the regarding Bot is opened, the App crashes. 
I already reset the App-Data on iOS, still crashing every time. I am using the current version (4.7.1) on iOS.

### Card JSON

```JSON
{
	"lang": "en",
	"$schema": "https://adaptivecards.io/schemas/adaptive-card.json",
	"version": "1.0",
	"type": "AdaptiveCard",
	"body": [{
		"type": "TextBlock",
		"text": "Hello World!",
		"wrap": true
		},{
		"type": "Image",
		"url": "https://cataas.com/cat",
		"size": "auto",
		"spacing": "none"}

]}
```

Edit: Corrected Boolean from True to true, Json was created from python-dict with botbuilder for python

### Sample Code

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by Meghana-MSFT** [comment]
@basti95 - We are also able to repro this issue, we have raised a bug for the same. We will keep you posted on the updates.

**Comment by RinkAttendant6** [comment]
Any update on this?  I tried switching to the [legacy MessageCard format](https://docs.microsoft.com/en-us/outlook/actionable-messages/message-card-reference) and it still crashes the channel for my team members who try to access it on iPhone

**Comment by Meghana-MSFT** [comment]
we do not have any update to share, we will let you know if there are any. Thanks.

**Comment by chienhsiang-hung** [comment]
Same problem on card ver >= 1.4

**Comment by Meghana-MSFT** [comment]
The root cause of this bug is a parsing issue due to the "lang" field on the json card. This has been fixed and the crash issue is resolved. We verified the fix in iOS Teams version 5.18.1. Could you please check once at your end.

**Comment by basti95** [comment]
Yes, i just tried the case again and the Problem seems to be fixed now! Glad that it finally got fixed! Great Work!Am 20.10.2023 um 20:48 schrieb Meghana-MSFT ***@***.***>:﻿
The root cause of this bug is a parsing issue due to the "lang" field on the json card. This has been fixed and the crash issue is resolved. We verified the fix in iOS Teams version 5.18.1. Could you please check once at your end.
